File: app/src/main/java/com/templates/kotlintemplates/utils/StorageUtils.java
package com.templates.kotlintemplates.utils;

import android.os.Environment;

public class StorageUtils {
    /**
     * Checks if the SD Card is mounted on the device.
     *
     * @deprecated use {@link #isSdCardMounted()}
     * **
     */
    public static boolean isSDCARDMounted() {
        String status = Environment.getExternalStorageState();
        if (status.equals(Environment.MEDIA_MOUNTED))
            return true;
        return false;
    }

    public static boolean isSdCardMounted() {
        String status = Environment.getExternalStorageState();
        if (status.equals(Environment.MEDIA_MOUNTED))
            return true;
        return false;
    }

}

File: app/src/main/java/com/templates/kotlintemplates/utils/StorageUtils.kt
package com.templates.kotlintemplates.utils

import android.os.Environment

object StorageUtils {
    /**
     * Checks if the SD Card is mounted on the device.

     */
    val isSDCARDMounted: Boolean
        @Deprecated("use {@link #isSdCardMounted()}\n      **")
        get() {
            val status = Environment.getExternalStorageState()
            if (status == Environment.MEDIA_MOUNTED)
                return true
            return false
        }

    val isSdCardMounted: Boolean
        get() {
            val status = Environment.getExternalStorageState()
            if (status == Environment.MEDIA_MOUNTED)
                return true
            return false
        }

}
